### ISSUE TITLE
Offset must be number?

### ISSUE THREAD

**Issue by ycmjason** [web-animations/web-animations-js#212]
According to the [mdn doc](https://developer.mozilla.org/en-US/docs/Web/API/Web_Animations_API/Keyframe_Formats), offset could be written in an array of numbers. Here is an example in the doc:

```js
element.animate({
  opacity: [ 0, 0.9, 1 ],
  offset: [ 0, 0.8 ], // Shorthand for [ 0, 0.8, 1 ]
  easing: [ 'ease-in', 'ease-out' ],
}, 2000);
```

I was trying to do exactly the same but it throws an error:

```
TypeError: Keyframe offsets must be numbers.
```

Is this a new feature that has not been polyfilled or am I expecting wrong thing from this lib?

**Comment by birtles** [comment]
Yes, arrays of offsets are a more recent addition to the spec ([added just under 2 years ago](https://github.com/w3c/web-animations/commit/f43ecdfbe5f35d243fcde01b372bf7de01a15110)) so I suspect the polyfill has not been updated to support that yet.
